test/test_sparse.py

```diff
@@ -49,6 +49,8 @@ def outer(self, *args, **kwargs):
 
 class TestSparse(TestCase):
 
+    value_dtype = torch.float64
```

Comment:
Why this change?

Reply:
@mruberry this was introduced for reusing `TestSparse` tests for complex dtype. See https://github.com/pytorch/pytorch/pull/50984/files#diff-bf009aed03d333c6a63253630fed4465ba5f7e025029fe2a22919759c38beb6dR3198-R3201